pt-BR/mail/chrome/messenger/preferences/downloads.dtd

```diff
@@ -1,24 +1,24 @@
-<!ENTITY askOnSave.label              "Attachments Folder">
-<!ENTITY alwaysAsk.label              "Ask me where to save every file">
+<!ENTITY askOnSave.label              "Pasta de anexos">
+<!ENTITY alwaysAsk.label              "Perguntar onde salvar cada arquivo">
 <!ENTITY alwaysAsk.accesskey          "A">
-<!ENTITY neverAsk.label               "Save all attachments to this folder:">
+<!ENTITY neverAsk.label               "Salvar todos os anexos nesta pasta:">
 <!ENTITY neverAsk.accesskey           "S">
-<!ENTITY desktop.label                "Desktop">
-<!ENTITY downloads.label              "My Downloads">
-<!ENTITY chooseFolderWin.label        "Browse...">
+<!ENTITY desktop.label                "Área de Trabalho">
+<!ENTITY downloads.label              "Downloads">
```

Comment:
@brunotiussi Actually this can be changed; example: in PT-PT the folder's name is "Transferências" although the way to arrive it's the same X:\User\Name\Downloads. Let's check with Sherman.